leveldb/src/main/java/org/iq80/leveldb/impl/Compaction.java

```diff
@@ -126,8 +126,10 @@ public boolean isTrivialMove()
     public static long totalFileSize(List<FileMetaData> files)
     {
         long sum = 0;
-        for (FileMetaData file : files) {
-            sum += file.getFileSize();
+        if (files != null) {
```

Comment:
Shouldn’t this be guarded in the caller? It should be an error for this method to be called with a null argument

Reply:
The code is not designed well, so I figured it was safer to do here.  This should really be `Optional<List<Files>>`, but I'd prefer to keep the changes minimal.